src/utils/css.utils.ts

```diff
@@ -1,5 +1,10 @@
-export const injectHeadCSS = () => {
-  let style: HTMLStyleElement | null = document.head.querySelector('style[stylo-editor]');
+export const injectHeadCSS = (rootNode: Node) => {
```

Comment:
probably worth renaming the function if it also inject css elsewhere than in head